temba/triggers/views.py

```diff
@@ -284,6 +286,36 @@ class Meta:
         fields = ("flow", "omnibox", "repeat_period", "repeat_days_of_week", "start_datetime")
 
 
+class ScheduleTriggerInBatchForm(ScheduleTriggerForm):
+    BATCH_INTERVAL = (
+        (5, _("5 minutes")),
+        (10, _("10 minutes")),
+        (15, _("15 minutes")),
+        (20, _("20 minutes")),
+        (25, _("25 minutes")),
+        (30, _("30 minutes")),
+    )
+
+    start_datetime = forms.DateTimeField(
```

Comment:
I think we don't need to redeclare `start_datetime` and `omnibox` on `ScheduleTriggerInBatchForm` since we already have it on `ScheduleTriggerForm`. Would be needed to redeclare in some attribute overwrite case.

Reply:
I had to redeclare here to make it a required field

Reply:
Oh, got it. Do we need to redeclare `omnibox` field? I mean, is there something that we need to overwrite?

Reply:
yes, I had to disable users selection, trying to focus on only group selection, so the override was to not set `"contacts": True`